src/dygraph-tickers.js

```diff
@@ -220,29 +220,36 @@ export var dateTicker = function(a, b, pixels, opts, dygraph, vals) {
 
 // Time granularity enumeration
 export var Granularity = {
-  SECONDLY: 0,
-  TWO_SECONDLY: 1,
-  FIVE_SECONDLY: 2,
-  TEN_SECONDLY: 3,
-  THIRTY_SECONDLY : 4,
-  MINUTELY: 5,
-  TWO_MINUTELY: 6,
-  FIVE_MINUTELY: 7,
-  TEN_MINUTELY: 8,
-  THIRTY_MINUTELY: 9,
-  HOURLY: 10,
-  TWO_HOURLY: 11,
-  SIX_HOURLY: 12,
-  DAILY: 13,
-  TWO_DAILY: 14,
-  WEEKLY: 15,
-  MONTHLY: 16,
-  QUARTERLY: 17,
-  BIANNUAL: 18,
-  ANNUAL: 19,
-  DECADAL: 20,
-  CENTENNIAL: 21,
-  NUM_GRANULARITIES: 22
+  MSLY: 0,
+  TWO_MSLY: 1,
+  FIVE_MSLY: 2,
+  TEN_MSLY: 3,
+  FIFTY_MSLY: 4,
+  HTH_MSLY: 5,
+  FHTH_MSLY: 6,
```

Comment:
Personally, I think there's no harm in spelling out "millisecond" instead of just using "ms". But, it's not my code 😉  (Also, as I mentioned in #776 I think it'd be interesting to explore alternative approaches that make this more customizable.)